statsmodels/tsa2/datasets/brockwell_davis_2002/__init__.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+from pytsa.datasets.brockwell_davis_2002.data.dowj import dowj
```

Comment:
it looks like these files are implemented below.  are you intentionally importing from a different package?

Reply:
Just a typo, thanks